control-plane/roles/gardener/tasks/extensions.yaml

```diff
@@ -69,7 +69,9 @@
   loop:
     - controller-deployment.yaml
     - controller-registration.yaml
-  when: gardener_extension_networking_cilium_enabled
+  when:
+    - gardener_extension_networking_cilium_enabled
+    - gardener_cert_management_issuer_email is defined
```

Comment:
I think this is the wrong place? I also meant adding an assert, which is at the beginning of tasks of this role in the `main.yaml`. The task is called `Check mandatory variables for this role are set`.

Reply:
Now it should be in the right place, I think.